pkgs/development/python-modules/nbclient/default.nix

```diff
@@ -1,27 +1,65 @@
-{ lib, buildPythonPackage, fetchPypi, pythonOlder,
-  async_generator, traitlets, nbformat, nest-asyncio, jupyter-client,
-  pytest, xmltodict, nbconvert, ipywidgets
+{ async_generator
+, buildPythonPackage
 , doCheck ? true
+, fetchFromGitHub
+, ipywidgets
+, jupyter-client
+, lib
+, nbconvert
+, nbformat
+, nest-asyncio
+, pytest-asyncio
+, pytestCheckHook
+, pythonOlder
+, traitlets
+, xmltodict
 }:
 
 buildPythonPackage rec {
   pname = "nbclient";
-  version = "0.5.13";
+  version = "0.6.2";
+  # As of version 0.6.2, there is both a setup.py and pyproject.toml. However,
+  # the pyproject.toml does not appear to be the main entry point.
+  format = "setuptools";
+
   disabled = pythonOlder "3.7";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "sha256-QMUsm148MfrsruafICs/U+ONfBxWPeD63enX7aD9r+g=";
+  src = fetchFromGitHub {
+    owner = "jupyter";
+    repo = pname;
+    rev = "refs/tags/v${version}";
```

Comment:
```suggestion
    rev = "v${version}";
```

Reply:
this is intentional. GitHub has a nasty bug when you happen to have both a branch and a tag that use the same name. therefore nixpkgs-update and other tools are transitioning to use refs/tags/foo